src/emmc.c

```diff
@@ -186,15 +187,20 @@ pu_emmc_setup_layout(PuFlash *flash,
             part->size = self->expanded_part_size;
         }
 
+        if (part->block_size > 0) {
+            part->size -= part->size % part->block_size;
+        }
+
         /* Increase size of logical partitions by 2 sectors for EBR.
          * The 2 sectors are removed during partition creation. */
         if (part->type == PED_PARTITION_LOGICAL) {
             part->size += 2;
         }
 
-        g_debug("%s: type=%d filesystem=%s start=%lld size=%lld offset=%lld expand=%s",
+        g_debug("%s: type=%d filesystem=%s start=%lld size=%lld offset=%lld \
+                block-size=%lld expand=%s",
```

Comment:
Multiline strings should be split like the following:
```
"one "
"two"
```
Using `\` will carry the indentation of the next line into the string, which we do not want.